devicon.json

```diff
@@ -8834,6 +8834,32 @@
             }
         ]
     },
+    {
+        "name": "remix",
+        "altnames": [
+            "remix.run"
+        ],
+        "tags": [
+            "framework",
+            "fullstack",
+            "web"
+        ],
+        "versions": {
+            "svg": [
+                "plain",
```

Comment:
Instead of **plain** and **plain-wordmark** use original and **original-wordmark**